problemsets-mat223/modules/module1.tex

```diff
@@ -479,7 +479,20 @@
 	Let $\vec x=\xhat-\yhat$, $\vec y=3\xhat-\yhat$, and $\vec r=2\xhat+2\yhat$. Is
 	$\vec r$ a linear combination of $\vec x$ and $\vec y$?
 
-	XXX Finish
+	Assume $\vec r$ is a linear combination of $\vec x$ and $\vec y$ for some unknown coefficients $\alpha_1$ and $\alpha_2$. Then
```

Comment:
No need to assume. Say, "By definition, $\vec r$ is a linear combination of $\vec x$ and $\vec y$ if there are scalars ..." "Thus we are looking for solutions to the vector equation ..."